examples_tests/50.MeshPacking/shader.vert

```diff
@@ -0,0 +1,17 @@
+#version 430 core
+
+layout(push_constant, row_major) uniform PushConstants
+{
+	mat4 vp;
+} pc;
+
+layout(location = 0) in vec3 vPos;
+layout(location = 3) in vec3 vNorm;

```

Comment:
you better override the MTL shader (like megatexture example does) because you'll need to fetch the per-vertex data by yourself.